### DISCUSSION TITLE
how can I use this package for angular.

### DISCUSSION THREAD

**Discussion by PRASHANTH990** [atlasmap/atlasmap#3383] [Q&A]
How can I use this package for angular? I am new to this Data mapper.

**Comment by igarashitm** [comment]
AtlasMap UI is based on React. There seems to be a way to embed React component in Angular app
https://www.google.com/search?q=react+component+in+angular&oq=react+component+in+angular&aqs=chrome.0.69i59i512j0i512j0i22i30l8.5636j0j7&sourceid=chrome&ie=UTF-8

Atlasmap component from @atlasmap/atlasmap package is the root component
https://github.com/atlasmap/atlasmap/blob/master/ui/packages/atlasmap-standalone/src/App.tsx#L45

**Reply by PRASHANTH990** [reply]
thank you
